Dockerfile.centos7

```
@@ -33,7 +32,13 @@ ARG NVIDIA_CONDA_LABEL=nvidia/label/cuda${CUDA_MAJORMINOR_VERSION}
 ARG RAPIDSAI_CONDA_LABEL=rapidsai/label/cuda${CUDA_MAJORMINOR_VERSION}
 ARG RMM_VERSION=0.7.*
 ARG PANDAS_VERSION=0.23.4
```

```suggestion
ARG PANDAS_VERSION=0.24.2
```
This might also have to be changed.